Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pie.yml

```diff
@@ -8,8 +8,6 @@
   - type: FlavorProfile
     flavors:
       - sweet
-  - type: Food
-    trash: FoodPlateTin
```

Comment:
shouldn't pies still create tins when eaten?

Reply:
pretty sure they still do i just removed that for tarts to not?

Reply:
cream pie still gives tin the others never visually had a tin